app/components/Settings/Preview/Preview.react.js

```diff
@@ -16,8 +16,8 @@ export default class Preview extends Component {
         const tableHeaders = columnnames.map(
             column => <th>{column}</th>
         );
-
-        const renderCell = cell => <td>{JSON.stringify(cell)}</td>;
+        const isObject = (arg) => typeof arg === 'object';
+        const renderCell = cell => <td>{isObject(cell) ? JSON.stringify(cell) : cell}</td>;
```

Comment:
is this just for the `geo_json` objects? any other `object` types? we should probably do this in our backend parsing too